project/src/test/scala/ScalastyleInspectionsGeneratorTest.scala

```diff
@@ -0,0 +1,141 @@
+import com.typesafe.config.ConfigFactory
+import org.scalastyle._
+import org.scalatest._
+
+import scala.io.Source._
+import scala.meta._
+import scala.xml._
+
+class ScalastyleInspectionsGeneratorTest extends FlatSpec with Matchers {
+
+  "extractInspections" should "successfully extract the inspections" in {
+    val inspections = XML.load(fromResource("scalastyle/definition.xml").bufferedReader) \\ "checker"
+    //XML.load(this.getClass.getResourceAsStream("scalastyle/definition.xml")) \\ "checker"
+    val docs = XML.load(fromResource("scalastyle/documentation.xml").bufferedReader) \\ "check"
+    // XML.load(this.getClass.getResourceAsStream("scalastyle/documentation.xml")) \\ "check"
```

Comment:
Personally I preffer to use `scala.io.Source` than `this.getClass.getResourceAsStream`.
However, I would rather just import **Source** and call it using `Source.fromResource`, also we should call bufferedReader with parentheses _()_ as stated in the [scaladoc](https://www.scala-lang.org/api/current/scala/io/BufferedSource.html#bufferedReader():java.io.BufferedReader).

Reply:
Ok, I'll stick to `scala.io.Source`, but I didn't import it because it collides with `scala.meta.Source` and I didn't want to rename them - I've imported `scala.io.Source.fromResource` instead, are you ok with that?

> we should call bufferedReader with parentheses () as stated in the scaladoc.

👍